app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java

```diff
@@ -22,6 +28,17 @@ public ParkingView(Activity activity, ActivityParkingBinding binding) {
         this.binding = binding;
     }
 
+    @Override
+    public void showDialog(ListenerDialogFragment listenerDialogFragment) {
```

Comment:
this is not correct, next time try to not modify previous changes.

Reply:
ok